### ISSUE TITLE
header file missing

### ISSUE THREAD

**Issue by pepa65** [LevitatingBusinessMan/snappy-cli#1]
The file snappy-c.h is not included in this repo, so can't be built.

**Comment by LevitatingBusinessMan** [comment]
Make sure you have the snappy library installed. It's called `libsnappy-dev` on debian and `snappy` on Arch.

**Comment by pepa65** [comment]
I now get (after some warnings):
```
/usr/bin/ld: /tmp/cctonD1r.o: in function `main':
snappy.c:(.text+0x231): undefined reference to `snappy_max_compressed_length'
/usr/bin/ld: snappy.c:(.text+0x254): undefined reference to `snappy_uncompressed_length'
/usr/bin/ld: snappy.c:(.text+0x284): undefined reference to `snappy_compress'
/usr/bin/ld: snappy.c:(.text+0x2be): undefined reference to `snappy_uncompress'
collect2: error: ld returned 1 exit status
make: *** [Makefile:5: build] Error 1
```

**Comment by LevitatingBusinessMan** [comment]
I thought you might run into that issue, I found it earlier today while testing on debian.

I pushed a commit that should fix it, but it just moves the `-lsnappy` flag further down the gcc arguments. Why it worked when compiling on Arch but not on Debian I am not sure.

Those warnings you mentioned only appear on debian, so my guess is that they have to do with the older snappy library used by debian.
